index.html

```diff
@@ -177,11 +176,19 @@ <h2>Multi-Post Stories</h2>
                     </ul>
                 </div>
                 <div class="grid-item5">
-                    <button class="buttonGR" type="button">See Project</button>
+                    <button data-modal-target="#displayWork" class="buttonGR" type="button">
+              See Project
+            </button>
                 </div>
             </div>
-            <button class="button1" type="button">See Project</button>
+            <button data-modal-target="#displayWork" class="button1" type="button">
+          See Project
+        </button>
         </div>
+
+        <div id="displayWork" class="displayWork"></div>
+        <div id="overlay"></div>
+
         <div class="work-portofolio-2">
             <div class="grid-container grid-container2">
                 <div class="grid-item1 ml"></div>
```

Comment:
- [ ] Kindly make sure you remove all the hard code for the projects list like in this  **[example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)** `i.e.: from LINE 159 - 278(index.html file)` then go in your `JavaScript` file and create an array of objects to store JavaScript objects containing the information for each project.